### ISSUE TITLE
Feature: push messages directly to a queue

### ISSUE THREAD

**Issue by nizmow** [jandelgado/rabtap#35]
I'd really love it if there were some type of `rabtap send` functionality, where we could push messages directly from a queue instead of to an exchange. Basically the reverse of the very nice `rabtap sub`. Apologies if this is already there, I've checked the docs but couldn't find anything.

Thanks!

**Comment by jandelgado** [comment]
Hi, 

as far as I know it's not possible to directly publish/send to a queue. The underlying API rabtap uses (https://github.com/streadway/amqp) does also not provide such an interface. The official RabbitMQ documentation states:

> The core idea in the messaging model in RabbitMQ is that the producer never sends any messages directly to a queue. Actually, quite often the producer doesn't even know if a message will be delivered to any queue at all.

(https://www.rabbitmq.com/tutorials/tutorial-three-python.html)

**However** since every queue is automatically bound to the so called `default` exchange (has the name ""), you can create a queue and publish your message to the `default` exchange with the queue's name as the routing key. 

Example:

First terminal:
```shell
$ rabtap queue create MYQUEUE
$ rabtap sub MYQUEUE
```
Second terminal:
```shell
$  echo "HELLO" | rabtap pub "" --routingkey MYQUEUE
```
After the last command, you should see the `HELLO` message being printed in the first terminal.

To sum it up: Feature can not be implemented.

**Comment by nizmow** [comment]
Thanks for your feedback and valuable advice! Silly of me not to check whether this was even supported by the broker in the first place.

The use case I had was a queue which is bound to multiple exchanges, so I can `sub` from the queue but can't `pub` the messages back to the queue. Of course, an exchange can also be bound to multiple exchanges, so I can simply create a single exchange in between and publish directly back to it. Works like a charm.
